README.md

```diff
@@ -86,7 +86,150 @@ There are several ways to execute the code cells in your notebook:
     panel. Here you can schedule your notebook to be executed once at some future
     time, or repeatedly at your specified interval.
 
-## 4. Save and Share
+## 4. Analyze the Results
+
+After running each cell of the notebook, the results will display. When we use pixiedust display() to create an interactive dataset, we are able to manipulate the data using tables, graphs, and charts.
+
+### Options for PixieDust Charts
+
+After running cell #3 "display(accidents)", we can see by clicking the "Options" button that we are able to manipulate the keys and values for the fields used in the chart:
```

Comment:
usually backticks around buttons like `Options` -- don't know yet if we overuse it -- but probably also use it for code like `display(accidents)`